src/main/java/hudson/plugins/build_timeout/impl/ElasticTimeOutStrategy.java

```diff
@@ -56,21 +65,37 @@ public ElasticTimeOutStrategy(int timeoutPercentage, int timeoutMinutesElasticDe
         this.numberOfBuilds = Integer.toString(numberOfBuilds);
     }
 
-    @DataBoundConstructor
+    @Deprecated
     public ElasticTimeOutStrategy(String timeoutPercentage, String timeoutMinutesElasticDefault, String numberOfBuilds) {
         this.timeoutPercentage = timeoutPercentage;
         this.timeoutMinutesElasticDefault = timeoutMinutesElasticDefault;
         this.numberOfBuilds = numberOfBuilds;
```

Comment:
Please call the new constructor passing the dafault value to `failSafeTimeoutDuration`.
That makes what you want to do clear.